packages/lib/testMountParameters/index.ts

```diff
@@ -61,6 +61,7 @@ export default (): MockParameters => {
       moveHandleActionButton: 0,
       plugin: {},
       language: '',
+      mapHasDimensions: null,
```

Comment:
Why use `null` at all? It seems to duplicate `false`. There is only one value change of interest, `falsy => true`. It does not help to change `null` to `false` since that only means the first render attempt did not run through, which is what usually happens. Is there a case I'm missing?

Reply:
I have adjusted this to be always a boolean 6710452